vscode/lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "node_modules/lerna/schemas/lerna-schema.json",
+  "version": "0.0.0",
+  "packages": [
+    "microsoft-kiota",
```

Comment:
I think the need to match what's in the package.json name. Aslo using more explicit names would probably help people running the commands

Reply:
I went back to read on it. It uses the folder names not what is in the package.json